xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v1/dto/account/BitfinexTradingFeeResponse.java

```diff
@@ -0,0 +1,60 @@
+package org.knowm.xchange.bitfinex.v1.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+
+@JsonIgnoreProperties({"maker_fees", "taker_fees"})
```

Comment:
this annotations seams pointless

Reply:
It is necessary to prevent jackson from throwing errors, not to mention the fact that it improves performance.

Reply:
but the BitfinexTradingFeeResponse has no "maker_fees", "taker_fees" properties
the nested BitfinexTradingFeeResponseRow class has them and they are not ignored there